helix-core/src/diagnostic.rs

```diff
@@ -1,4 +1,6 @@
 //! LSP diagnostic utility types.
+use core::fmt;
```

Comment:
very minor nit but might as well use `std::fmt` for this instead, no?

Reply:
yeah I often endup messing up imports because I select the wrong completion item by accident. I found out ra will put the right item on top 99% of the time but we mess up completion ordering so the order is kind of random right now. Fixing that is part of my larger changes :P